www/SimpleCameraPreview.js

```diff
@@ -28,8 +28,18 @@ SimpleCameraPreview.torchSwitch = function (options, onSuccess, onError) {
   exec(onSuccess, onError, PLUGIN_NAME, "torchSwitch", [options]);
 };
 
+SimpleCameraPreview.switchToUltraWideCamera = function (options, onSuccess, onError) {
```

Comment:
do we make it generic like
switchCameraTo(options)

//calls
switchCameraTo("wide-angle");
switchCameraTo("default");